src/index.js

```diff
@@ -978,6 +978,30 @@ class ServerlessAppsyncPlugin {
         }
       }
 
+      if (config.sync) {
+        if (tpl.sync === true) {
+          // Use defaults
+          Properties.SyncConfig = {
+            ConflictDetection: config.sync.ConflictDetection || 'VERSION',
+          };
+        } else if (typeof tpl.sync === 'object') {
+          if (tpl.sync.lambda === true) {
```

Comment:
@hounded I don´t think this ´lambda´ true/false config is necessary. We can rely on the `ConflictHandler` value.

Reply:
Agreed